rre-search-platform/rre-search-platform-solr-impl/src/main/java/io/sease/rre/search/api/impl/ApacheSolr.java

```diff
@@ -85,14 +112,14 @@ public void afterStart() {
 
     @Override
     public void beforeStop() {
-        ofNullable(proxy).ifPresent(solr -> {
-            try {
-                solr.deleteByQuery("*:*");
-                solr.commit();
-            } catch (final Exception exception) {
-                exception.printStackTrace();
-            }
-        });
+//        ofNullable(proxy).ifPresent(solr -> {
+//            try {
+//                solr.deleteByQuery("*:*");
+//                solr.commit();
+//            } catch (final Exception exception) {
+//                exception.printStackTrace();
+//            }
+//        });
     }
```

Comment:
Can I ask you the reason of this comments? On top of that, I usually avoid to put line commented code in the main branch. I'm sure you had a reason for that, please expand a bit and then we will remove this fragment.

Reply:
Thanks for pointing these out. I've made some further changes in the ApacheSolr class and its Maven plugin so that the temp data directory is handled better, and should be deleted after processing again. The code above is not commented, but is in an `if` block.